File: fixtures/kotlin/inherits.kt
interface Describe {
    fun describe(): String {
        return "thing"
    }
}

open class Base(x: Int) {
    fun area(): Int {
        return 1
    }
}

class Circle : Base(1), Describe {
    override fun describe(): String {
        return "circle"
    }
}

class Plain {
    fun value(): Int {
        return 1
    }
}

class Ranked : Comparable<Ranked> {
    override fun compareTo(other: Ranked): Int {
        return 0
    }
}

object Registry : Describe {
    fun count(): Int {
        return 0
    }
}

File: fixtures/python/inherits.py
class Describe:
    def describe(self):
        return "thing"


class Base:
    def area(self):
        return 1


class Circle(Base, Describe):
    def describe(self):
        return "circle"


class Plain:
    def value(self):
        return 1


class Ranked(Comparable[int]):
    def rank(self):
        return 0


class Registry(Describe, metaclass=type):
    def count(self):
        return 0
